site/_posts/2021-02-16-orbitvu-fusionauth-story.md

```diff
@@ -0,0 +1,88 @@
+
+**Dan:** What internal systems have you hooked up to FusionAuth? Were there any challenges doing so?
+
+**Maciej:** We've integrated our helpdesk system (Freshdesk) and our manuals site (Screensteps). This was rather easy due to the standards provided by FusionAuth. We've also tested the integration with our ERP system which is an open-source solution Odoo. The proof-of-concept implementation we did work with FusionAuth using the OAuth2 protocol but before going further we need to resolve some other non-technical issues.
+
+**Dan:** What problems did we solve for you? And how were you solving them before FusionAuth?
+
+**Maciej:** Before Orbitvu has adopted FusionAuth we've used an SSO solution based on CAS (Central Authentication Service). It worked well for a long time and mostly for the traditional, monolithic web applications but in the era of SPA, React Native, microservices, and Kubernetes it became obvious that the technological debt regarding our Auth system is too big. This is where FusionAuth comes in. It has brought us solutions matching the newest trends like OAuth2 + OIDC or 2FA and has opened a whole bunch of new possibilities in terms of possible architectural solutions and technologies we were able to use. At the same time, it became possible to integrate our system with other providers thanks to the OAuth2 or SAML.
+
+**Dan:** Why did you choose FusionAuth over the competition?
+
+**Maciej:** We've reviewed multiple solutions like Auth0, Okta, AWS Cognito, and MiniOrange but we decided to use FusionAuth due to its flexibility, great documentation, and competitive pricing. I'd say that flexibility (also understood as customizability) of the solution is where FusionAuth stands out. It was important to us that we can use FusionAuth both on-premise (including docker and Kubernetes) as well as a hosted, hassle-free Saas version. Thanks to the docker images and FusionAuth's Kickstart system we could easily integrate FusionAuth into our development process to build reproducible development environments.
+
+Another thing about flexibility, where the competition is far behind, is the possibility to customize the look and feel of almost every aspect of the FusionAuth using themes. By the way, personally I don't like FreeMarker's syntax but it does the job! It was critical to us to be able to have our own layout applied to login pages and emails.
+
+One more important factor that we considered was the way to migrate from our existing solution to the new one. Again, FusionAuth's way of doing this fitted our needs perfectly and made it possible to migrate user password hashes from the Django-based system to FusionAuth flawlessly (initially we've created our own password encryptor to handle this but turned out the built ones were enough).
```

Comment:
~One more~ Another important factor that we considered was the ~way to migrate~ **migration** from our ~existing~ **old** solution to ~the new one~ **FusionAuth.**

Reply:
Again, FusionAuth ~'s way of doing this~ fit ~ted~ our needs perfectly and made it possible to migrate user password hashes ~from the Django-based system to FusionAuth~ flawlessly